en/docs/using-wso2-identity-server/creating-users-using-the-ask-password-option.md

```diff
@@ -46,72 +44,68 @@ Follow the instructions given below to configure this feature.
     Ensure that the new Identity Listeners with
     `         orderId=95        ` and `         orderId=97        ` are set
     to **true** in the
-    `         <IS_HOME>/repository/conf/identity/identity.xml        ` file.
+    `         <IS_HOME>/repository/conf/deployment.toml       ` file.
     
     By default, the properties are configured as shown below. Therefore, if
     you have not changed the default configurations, you can skip this step.
     
     ``` java
-    <EventListener type="org.wso2.carbon.user.core.listener.UserOperationEventListener" name="org.wso2.carbon.identity.mgt.IdentityMgtEventListener" orderId="50" enable="false"/>
-    <EventListener type="org.wso2.carbon.user.core.listener.UserOperationEventListener" name="org.wso2.carbon.identity.governance.listener.IdentityMgtEventListener" orderId="95" enable="true" />
-    <EventListener type="org.wso2.carbon.user.core.listener.UserOperationEventListener" name="org.wso2.carbon.identity.governance.listener.IdentityStoreEventListener" orderId="97" enable="true">
-```
+    [event.default_listener.identity_mgt]
+    priority= "50"
+    enable = false
+    [event.default_listener.governance_identity_mgt]
+    priority= "95"
+    enable = true
+    [event.default_listener.governance_identity_store]
+    priority= "97"
+    enable = true
+    ```
 
 
 Follow the steps given below to configure WSO2 IS to enable the ask
 password feature:
 
 1.  Make sure the following configuration is set (uncommented) in the
-    `           <IS_HOME>/repository/conf/identity/identity.xml          `
+    `           <IS_HOME>/repository/conf/deployment.toml          `
     file under `           <EmailVerification>          ` element below
     `           <Server>          ` element to set the redirection URL
```

Comment:
this instruction should change according to the toml config model. I think the following configs should be added into the toml file and not uncommented?